javascripts/discourse/components/dc-categories-only.js.es6

```diff
@@ -1,8 +1,11 @@
 import { openNewCategoryModal } from "discourse/routes/discovery-categories";
 import { computed } from "@ember/object";
 import Component from "@ember/component";
+import { inject as service } from "@ember/service";
 
 export default Component.extend({
+  router: service(),
```

Comment:
@duranmla nice, but I don't understand how this works to be honest. Do you mind providing a bit of context?